lib/actions/actionsBuilder.js

```diff
@@ -0,0 +1,30 @@
+const AddLinkToProjectAction = require('./AddLinkToProjectAction');
+const assert = require('assert');
+
+class ActionsBuilder {
+  constructor(){
+    this.commandMap = {
+      'add': new AddLinkToProjectAction,
```

Comment:
Probably want `()` on that class name. Not required per-se but it’s pretty much expected